drivers/espi/Kconfig.it8xxx2

```diff
@@ -21,6 +21,81 @@ config ESPI_PERIPHERAL_HOST_IO
 config ESPI_PERIPHERAL_DEBUG_PORT_80
 	default y
 
+config ESPI_PERIPHERAL_EC_HOST_CMD
+	default y
+
+choice IT8XXX2_H2RAM_HC_SIZE_CHOICE
+	prompt "choose a H2RAM space for ec host command"
+	depends on ESPI_PERIPHERAL_EC_HOST_CMD
+config IT8XXX2_H2RAM_HC_SIZE_256
+	bool "256"
+config IT8XXX2_H2RAM_HC_SIZE_16
```

Comment:
please a blank line on top, also on the choices below.

Reply:
Done.